package.json

```diff
@@ -22,21 +22,21 @@
   },
   "homepage": "https://github.com/wikimedia/restbase-mod-table-sqlite",
   "dependencies": {
-    "bluebird": "~2.8.2",
+    "bluebird": "^2.10.0",
```

Comment:
Let's keep it at 2.8.2 for now. We know it works in prod, and all of the other RESTBase components use it as well.